Automattic/wp-calypso
# wp-admin Site Default: Card tweaks

From pdKhl6-2cJ-p2#comment-5186

## Proposed Changes

Couple of tweaks to the admin interface style card:
* Changes the description text to clarify that the setting is site-wide.
* Deep links to the relevant documentation.

![CleanShot 2024-01-12 at 15 28 12@2x](https://github.com/Automattic/wp-calypso/assets/36432/b11b47db-e838-41ee-8d26-d72c3af05462)

## Testing Instructions

1. Create a Business site and take it Atomic.
2. Navigate to Hosting Configuration and verify the Admin interface style card looks as expected.
